package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "knex",
-  "version": "1.0.0",
+  "version": "0.95.14",
```

Comment:
why did you change the version? we can backport to 0.95 branch later, but master is for 1.0.0

Reply:
I merged to the 0.95.14 tag to update this fork because our application uses this fork for now. However, that tag has 1.0.0 in the package.json file for some reason. That version inconsistency was causing some weird sub-dependency issues that was causing type errors so I did this to fix it.

Reply:
In retrospect, perhaps our application should be looking at a different branch so as to not interfere with this PR. Apologies for the mess. I had forgotten this PR existed while I was trying to get our app working.

Reply:
ah, I see. No worries :)